client/src/components/App.js

```diff
@@ -32,6 +33,7 @@ const App = (props) => {
         <Route exact path="/users/new" component={RegistrationForm} />
         <Route exact path="/user-sessions/new" component={SignInForm} />
         <Route exact path="/:id" component={TripShow} />
+        <Route exact path="/:id/edit" component={EditForm} />
```

Comment:
just noticing this now, but lines 35 and 36 need an update for their paths 
`/trips/:id`
`trips/:id/edit` 

This will put them in line with restful best practices